<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactView</class>
 <widget class="QDialog" name="ContactView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
     <item row="1" column="0" colspan="2">
      <widget class="QTreeWidget" name="twContPhones">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>4</number>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">3</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">4</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QFrame" name="frContView">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="2">
         <widget class="QToolButton" name="btContEdit">
          <property name="text">
           <string>Edit</string>
          </property>
          <property name="icon">
           <iconset resource="taki.qrc">
            <normaloff>:/icons/edit.png</normaloff>:/icons/edit.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QToolButton" name="btContBack">
          <property name="text">
           <string>Back</string>
          </property>
          <property name="icon">
           <iconset resource="taki.qrc">
            <normaloff>:/icons/arrow-left.png</normaloff>:/icons/arrow-left.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QToolButton" name="btContDelete">
          <property name="text">
           <string>Delete</string>
          </property>
          <property name="icon">
           <iconset resource="taki.qrc">
            <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lbContactPhoto">
       <property name="text">
        <string>Photo</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lbContactDisplayName">
       <property name="text">
        <string>DisplayName</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="taki.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>twContPhones</sender>
   <signal>itemClicked(QTreeWidgetItem*,int)</signal>
   <receiver>ContactView</receiver>
   <slot>onPhoneSelected(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btContBack</sender>
   <signal>clicked()</signal>
   <receiver>ContactView</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onPhoneSelected(QTreeWidgetItem*,int)</slot>
 </slots>
</ui>
